docs/blog/oras-0.14-and-future.md

```diff
@@ -0,0 +1,107 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_[Feynman Zhou](https://twitter.com/FeynmanZhou), CNCF Ambassador, ACR Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the ORAS CLI recently. ORAS v0.14 introduces four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
+
+## What's new in ORAS 0.14
+
+<figure>
+  <img src="/blog/oras-0.14-and-future/carbon.svg" />
+</figure>
+
+
+Please see the [Release Notes](https://github.com/oras-project/oras/releases/tag/v0.14.0) for details.
+
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provides new functions to enable developers to build your own OCI client tool.
+
+As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signing. 
+
+This blog will demonstrate how to use ORAS CLI v0.14 to copy an image from a validated artifact registry to a personal conintainer registry, then attach a SBOM to it and discover the reference in a tree graph. 
+
+![ORAS workflow](oras-0.14-and-future/Screenshot_1.png)
+
+> Note: we will use MAR (Microsoft Artifact Registry) and ACR (Azure Container Registry) for demonstration purpose only. There will be another [blog posts](https://github.com/oras-project/oras-www/issues/54) to demonstrate how to use ORAS with Amazon ECR and Google GAR soon. 
+
+## Install ORAS 0.14
+
+Install the latest release of ORAS on a Linux machine:
+
+```
+curl -LO https://github.com/oras-project/oras/releases/download/v0.14.1/oras_0.14.1_linux_amd64.tar.gz
+mkdir -p oras-install/
+tar -zxf oras_0.14.1_*.tar.gz -C oras-install/
+mv oras-install/oras /usr/local/bin/
+rm -rf oras_0.14.1_*.tar.gz oras-install/
+```
+
+> Note: You can also refer to the [installation guide](https://oras.land/cli/) for other different platforms.
+
+## Copy an image from Registry A to Registry B
+
+In this demo, assume all images are validated in MAR, so I will use ORAS to copy the container image from MAR to my personal repository of ACR. You can use your prefered container registry with ORAS.
```

Comment:
Not sure what all images are validated in MAR implies.  
Suggest:

> In this demo we'll use ORAS to copy the container image from the public MAR registry to my private ACR registry. You can use your preferred container registry with ORAS.

nit, on preferred spelling